test/formatters/msbuildFormatterTests.ts

```diff
@@ -16,7 +16,7 @@
 
 import * as ts from "typescript";
 
-import {IFormatter, RuleFailure, TestUtils} from "../lint";
+import {IFormatter, RuleFailure, RuleLevel, TestUtils} from "../lint";
```

Comment:
Yeah, breaking means that this will have to wait for the v5.0 release. Can you put something in CHANGELOG.md and comment it out? Something like:

```
<!--
v5.0.0
---

* **BREAKING CHANGES**
    *  The severity level is now configurable and the default severity may have changed:
        - "msbuild" was "warning", but now defaults to "error"
        - "vso" ......
-->